features/363-glossary/glossary.md

```diff
@@ -0,0 +1,53 @@
+Glossary
+========
+
+In this document, we define a list of terms to be used in communication.
+
+Objective
+---------
+
+In various documents, we will use different terms to refer to concepts in Skygear:
+- Documentation
+- Technical discussions
+
+It would be clearer to use widely agreed terms to refer to specific concepts.
+
+Glossory
+--------
+- **Skygear Cluster**: the Skygear services (Gears, Gateway, and Controller), hosted on Kubernetes.
+- **Skygear Controller**: the management interface of Skygear Cluster.
+- **Skygear Portal**: the web management interface of Skygear Cluster.
```

Comment:
Not sure if we should also add `skycli` to this list. If we do, `: the command line interface of Skygear Cluster`

Reply:
Added skycli to glossary.